### PR TITLE
Feature/#dev 307/tech page open source

### PR DESCRIPTION
**Pull request by smoliver**
screen shot 2018-06-24 at 8 08 06 pm
screen shot 2018-06-24 at 8 07 49 pm
screen shot 2018-06-22 at 3 59 02 pm
screen shot 2018-06-24 at 8 08 30 pm
screen shot 2018-06-24 at 8 08 40 pm
screen shot 2018-06-24 at 8 08 22 pm

### CODE REVIEW

**Comment by tavakyan** [comment]
There is a typo on "Convertible debpt raise".

**Comment by smoliver** [comment]
Edits:
* Timeline: Debpt -> Debt 
* Papers: sub-header and description text

File: web-app/modules/timeline-data/data.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  phases: [
+    {
+      name: 'phase I',
+      milestones: [
+        {
+          name: 'Release of White Paper v1.1',
+          date: 'Apr 26, 2018'
+        },
+        {
+          name: 'Convertible Debpt Raise of 450k',
```
Comment (tavakyan) [inline web-app/modules/timeline-data/data.js]: Debt*

File: web-app/sections/technology/sections/timeline/index.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import PropTypes from 'prop-types'
+import TimelineData from 'modules/timeline-data'
+import s from './index.scss'
+
+Timeline.propTypes = {
+  phases: PropTypes.array.isRequired
+}
+
+function Timeline({ phases, ...rest }) {
+  // console.log(phases)
```
Comment (tavakyan) [inline web-app/sections/technology/sections/timeline/index.js]: Not sure if you want to leave this comment in or if you use it for debugging, but if its pretty stable seems safe to delete.

**Review by tavakyan — DISMISSED** [review]
Looks good except two minor things

File: web-app/modules/inject/index.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export default function inject(data, Component) {
```
Comment (davesag) [inline web-app/modules/inject/index.js:3]: any reason why you didn't do

```
export default (data, Component) => props => <Component {...props} {...data} />
```

**Comment by davesag** [comment]
In general, unless a function needs access to its own `this`, I prefer to use arrow functions to classical functions.

**Comment by smoliver** [comment]
@davesag  As a practice I tend to export named functions / classes, just for debugging purposes.

**Review by davesag — APPROVED** [review]

**Pull request merged**